ha/core/system_health/bootstrap/json_kv_convertor.py

```diff
@@ -0,0 +1,143 @@
+#!/usr/bin/env python3
+
+# Copyright (c) 2021 Seagate Technology LLC and/or its Affiliates
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+
+
+import json
+import re
+
+from cortx.utils.log import Log
+from ha.core.system_health.system_health_exception import InvalidHealthDataException
+from ha.core.system_health.bootstrap.health_event_generator import HealthEventGenerator
+from ha.execute import SimpleCommand
+from ha.core.system_health.const import NODE_HEALTH_KV_FILE
+
+class KVGenerator:
```

Comment:
Lets discuss with Monitor/Foundation team and see if something available already for this.

Reply:
Yes. This point is already mentioned in the doc LLD: System Health Bootstrap [TBD]

Reply:
We have added this item to the DOD of EOS-18277, so we are good.